js/core/devices.js

```diff
@@ -295,15 +305,13 @@ class Devices {
     }
 
     _fromUA(ua) {
-        let config;
-
-        each(uaParsers, (platform, parser) => {
-            config = parser(ua);
-            return !config;
-        });
+        for(let idx = 0; idx < UA_PARSERS_ARRAY.length; idx += 1) {
```

Comment:
let's rename idx to index

Reply:
Let's discuss this comment with the team separately.
From my point of view, it's a matter of taste, so we need the team's code style agreement here.

Reply:
Yeah, it's definitely a matter of taste, but i'm personally into index/i